tcms/bugs/templates/bugs/search.html

```diff
@@ -82,7 +82,7 @@
 
             <label class="col-md-1 col-lg-1" for="id_status">{% trans "Open" %}</label>
             <div class="col-md-3 col-lg-3">
-                <input class="bootstrap-switch" id="id_status" type="checkbox" checked>
+                <input class="bootstrap-switch" id="id_status" type="checkbox" checked data-on-text="Yes" data-off-text="No">
```

Comment:
Wrap the values of the data-on-text and data-off-text attributes with trans like it's done on the line above. It's needed for internationalization.